### PR TITLE
Issue #2: Multi-line output (log/list)

### PR DESCRIPTION
**Pull request by g4z**
Hi, See issue #2. This works for me. Please merge if you want. Regards!

### CODE REVIEW

File: example.js [hunk 1]
```diff
@@ -3,14 +3,19 @@ var Ora = require('./');
...
+	spinner: process.argv[2],
+	split: true
 });
 
 spinner.start();
 
 setTimeout(() => {
-	spinner.color = 'yellow';
-	spinner.text = 'Loading rainbows';
+	spinner.setColor('yellow');
```
Comment (SamVerschueren) [inline example.js]: You can use [`Object.defineProperty`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty) to do the same thing.

```js
Object.defineProperty(this, 'color', {
  set: function(color) {
    // do whatever you want to do when `spinner.color` is set.
  }
});
```

File: example.js [hunk 2]
```diff
@@ -3,14 +3,19 @@ var Ora = require('./');
...
+	spinner.setText('Loading rainbows');
```
Comment (SamVerschueren) [inline example.js]: same as above

File: index.js [hunk 3]
```diff
@@ -3,6 +3,7 @@ var chalk = require('chalk');
 var cliCursor = require('cli-cursor');
 var cliSpinners = require('cli-spinners');
 var objectAssign = require('object-assign');
+var EOL = require('os').EOL;
```
Comment (SamVerschueren) [inline index.js]: use `\n`

File: index.js [hunk 4]
```diff
@@ -35,8 +37,28 @@ function Ora(options) {
 	this.id = null;
 	this.frameIndex = 0;
 	this.enabled = (this.stream && this.stream.isTTY) && !process.env.CI;
+	this.completed = '√';
```
Comment (SamVerschueren) [inline index.js]: remove

File: index.js [hunk 5]
```diff
@@ -35,8 +37,28 @@ function Ora(options) {
...
+			this.complete();
+		}
+	}
+	this.text = text;
+};
+
+Ora.prototype.complete = function () {
+	this.clear();
+	var tick = chalk.green(this.completed);
+	this.render(tick + ' ' + this.text + EOL);
```
Comment (SamVerschueren) [inline index.js]: use `[log-symbols](https://github.com/sindresorhus/log-symbols).success` instead.

**Comment by SamVerschueren** [comment]
Gave some feedback. Not sure if @sindresorhus wants this though as the library's description says

> Elegant terminal spinner

This behaviour might not really fit into the idea of the library.

**Comment by sindresorhus** [comment]
Sorry about delay @g4z. I'm strapped for time. I haven't decided yet. Need to find some time this week to review the PR and think about the best API for something like this. But I do agree with the PR feedback from @SamVerschueren.

**Comment by sindresorhus** [comment]
This is now possible with: https://github.com/SamVerschueren/listr

**Pull request closed without merging**